[2024-05-05T10:41:54.153+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T10:41:54.166+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sample.transform_to_db scheduled__2024-05-05T09:00:00+00:00 [queued]>
[2024-05-05T10:41:54.174+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sample.transform_to_db scheduled__2024-05-05T09:00:00+00:00 [queued]>
[2024-05-05T10:41:54.174+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T10:41:54.190+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transform_to_db> on 2024-05-05 09:00:00+00:00
[2024-05-05T10:41:54.200+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=98930) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-05T10:41:54.199+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sample', 'transform_to_db', 'scheduled__2024-05-05T09:00:00+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmphv8xeib0']
[2024-05-05T10:41:54.200+0000] {standard_task_runner.py:63} INFO - Started process 98932 to run task
[2024-05-05T10:41:54.201+0000] {standard_task_runner.py:91} INFO - Job 229: Subtask transform_to_db
[2024-05-05T10:41:54.243+0000] {task_command.py:426} INFO - Running <TaskInstance: sample.transform_to_db scheduled__2024-05-05T09:00:00+00:00 [running]> on host 5bc5768c5236
[2024-05-05T10:41:54.322+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sample' AIRFLOW_CTX_TASK_ID='transform_to_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-05T09:00:00+00:00'
[2024-05-05T10:41:54.323+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T10:41:54.681+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/my_dag.py:128 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-05-05T10:42:01.949+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-05T10:42:01.949+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T10:42:01.957+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=sample, task_id=transform_to_db, execution_date=20240505T090000, start_date=20240505T104154, end_date=20240505T104201
[2024-05-05T10:42:01.988+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T10:42:02.002+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05T10:42:02.008+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
